workspaces/core/src/combobox/Combobox.tsx

```diff
@@ -1,180 +1,544 @@
+        multiple={multiple !== undefined}
+        counter={counter}
+        {...rest}
+      >
+        <Input
+          open={open}
+          placeholder={placeholder}
+          onChange={onChange}
+          onQueryChange={onQueryChange}
+          displayValue={displayValue}
+        />
+        <Button open={open}>
+          {children}
+        </Button>
+      </Listbox.Button>
+    </Listbox>
+  );
+};
+
+const MultiSelect = ({
+  open,
+  label,
+  placeholder,
+  children,
+  className,
+  multiple = true,
+  counter,
+  onChange,
+  onQueryChange,
+  displayValue,
+  ...rest
+}: WithChildren<SelectProps & InputProps>) => {
+  const { size, popper, disabled } = useComboboxContext('Combobox.MultiSelect');
+
+  return (
+    <Listbox>
+      {label && (
+        <SelectButton.Label labelSize={size} idDisabled={disabled}>
+          {label}
+        </SelectButton.Label>
+      )}
+      <Listbox.Button
+        open={open}
+        as={Trigger}
+        ref={popper?.setAnchor}
+        className={className}
+        multiple={multiple !== undefined}
+        counter={counter}
+        {...rest}
+      >
+        {counter !== undefined && counter > 0 && (
+          <Counter counter={counter} />
+        )}
+        <Input
+          open={open}
+          placeholder={placeholder}
+          onChange={onChange}
+          onQueryChange={onQueryChange}
+          displayValue={displayValue}
+        />
+        <Button open={open}>
+          {children}
+        </Button>
+      </Listbox.Button>
+    </Listbox>
+  );
+};
+
+const InsetSelect = ({
+  open,
+  label,
+  placeholder,
+  children,
+  className,
+  multiple,
+  counter,
+  onChange,
+  onQueryChange,
+  displayValue,
+  ...rest
+}: WithChildren<SelectProps & InputProps>) => {
+  const { popper } = useComboboxContext('Combobox.Select');
+
+  return (
+    <Listbox>
+      <Listbox.Button
+        open={open}
+        as={Trigger}
+        ref={popper?.setAnchor}
+        className={className}
+        multiple={multiple !== undefined}
+        counter={counter}
+        innerLabel={true}
+        {...rest}
+      >
+        <InsetInput
+          open={open}
+          label={label}
+          placeholder={placeholder}
+          onChange={onChange}
+          onQueryChange={onQueryChange}
+          displayValue={displayValue}
+        />
+        <Button open={open}>
+          {children}
+        </Button>
+      </Listbox.Button>
+    </Listbox>
+  );
+};
+
+const InsetMultiSelect = ({
+  open,
+  label,
+  placeholder,
+  children,
+  className,
+  multiple = true,
+  counter,
+  onChange,
+  onQueryChange,
+  displayValue,
+  ...rest
+}: WithChildren<SelectProps & InputProps>) => {
+  const { popper } = useComboboxContext('Combobox.MultiSelect');
+
+  return (
+    <Listbox>
+      <Listbox.Button
+        open={open}
+        as={Trigger}
+        ref={popper?.setAnchor}
+        className={className}
+        multiple={multiple !== undefined}
```

Comment:
if `multiple = true,` always by default, does this check make sense? And at all, could multiple be false in this subcomponent?